### ISSUE TITLE
Support for linked git worktrees broken in 1.2.7

### ISSUE THREAD

**Issue by markrian** [evilmartians/lefthook#417]
### :wrench: Summary

Running `lefthook install` in a repository that is a linked worktree (not the main worktree) causes an error like:

```
$ lefthook install
Error: mkdir /home/markrian/my-linked-repo/home/markrian/my-main-repo/.git/info: no such file or directory
```

### Lefthook version

1.2.7 c88a009cfa095e97ce47f8c5d5e1db5e6c844341

### Steps to reproduce

1. In a main worktree (any normal git repo) create a linked worktree e.g., `git worktree add ~/path/to/new/repo`
2. Change directory to new repo (e.g., `~/path/to/new/repo`)
3. Run `lefthook install`
4. Observe an error like:

```
$ lefthook install
Error: mkdir /home/markrian/path/to/new/repo/home/markrian/my-main-repo/.git/info: no such file or directory
```

If you try the same with lefthook 1.2.6, it succeeds without error.

If you run `lefthook install` from the _main_ worktree, it also succeeds without error.

### Expected results

`lefthook install` should work without error.

### Actual results

There's an error.

### Possible Solution

I suspect https://github.com/evilmartians/lefthook/pull/414 is the source of this breakage, since it adds a `mkdir` call.

### Screenshots (if appropriate)

n/a

**Comment by mrexox** [comment]
Hey! Thank you for the issue. I will take a look and try to fix it today-tomorrow.

**Comment by mrexox** [comment]
Than you a lot for a clean bug description! I will release the fix with version 1.2.8 soon
